README.md

```diff
@@ -4,7 +4,13 @@
 [Italiano](https://pnpm.io/it/) |
 [Português Brasileiro](https://pnpm.io/pt/)
 
-![](https://i.imgur.com/qlW1eEG.png)
+<h1>
+  <picture>
+    <source media="(prefers-color-scheme: light)" srcset="https://i.imgur.com/qlW1eEG.png">
+    <source media="(prefers-color-scheme: dark)"  srcset="https://i.imgur.com/qlW1eEG.png">
+    <img src="https://i.imgur.com/qlW1eEG.png" alt="pnpm">
+  </picture>
+</h1>
```

Comment:
Fixed the image not displaying.

- [Before](https://github.com/pnpm/pnpm/blob/058b540c8f47dabf09ae30a171a808183767f062/README.md) on [♻️ Remove logo link](https://github.com/pnpm/pnpm/pull/8721/commits/058b540c8f47dabf09ae30a171a808183767f062) 058b540c8f47dabf09ae30a171a808183767f062
- [After](https://github.com/pnpm/pnpm/blob/8f437f0ab1b4bee19dca3478ca239d625da405d9/README.md) on [🐛 Fix unvisible logo](https://github.com/pnpm/pnpm/pull/8721/commits/8f437f0ab1b4bee19dca3478ca239d625da405d9) 8f437f0ab1b4bee19dca3478ca239d625da405d9